### PR TITLE
Drawing Classifier Evaluation

### PR DESCRIPTION
**Pull request by abhishekpratapa**

### CODE REVIEW

File: src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py [hunk 1]
```diff
@@ -688,8 +693,141 @@ def evaluate(self, dataset, metric='auto', batch_size=None, verbose=True):
...
+            return _reduce(lambda x, y: x + (y*math.log(1/y, 2) if y > 0 else 0) , probs, 0) / math.log(len(probs),2)
+
+        def confidence(probs):
+            return max(probs)
+
+        def relative_confidence(probs):
+            lp = len(probs)
+            return probs[lp-1] - probs[lp-2]
+
+        def get_confusion_matrix(extended_test, labels):
```
Comment (znation) [inline src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py]: This code seems duplicated with the image_classifier implementation. These functions also only seem relevant for `explore`, not the rest of the evaluation. Can we refactor them into a common .py file and share the implementation among both toolkits?

File: src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py [hunk 2]
```diff
@@ -676,94 +676,16 @@ def evaluate(self, dataset, metric='auto', verbose=True, batch_size=64):
...
-            return vecs
-
-        def l2Dist(v1, v2):
-            dist = 0
-            for i in range(0, len(v1['pos'])):
-                dist +=  math.pow(v1['pos'][i] - v2['pos'][i], 2)
-            return math.pow(dist, 0.5)
-
-
+        # TODO: Imports here
```
Comment (srikris) [inline src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py]: What is this TODO :)

File: src/unity/python/turicreate/toolkits/_evaluate_utils.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
+def entropy(probs):
+    return _reduce(lambda x, y: x + (y*math.log(1/y, 2) if y > 0 else 0) , probs, 0) / math.log(len(probs),2)
```
Comment (srikris) [inline src/unity/python/turicreate/toolkits/_evaluate_utils.py]: Please file an issue to move this code to C++ eventually.

File: src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py (hunk 2 above)
Comment (abhishekpratapa) [reply src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py]: * A personal TODO, I didn't remove. Removed it!

File: src/unity/python/turicreate/toolkits/_evaluate_utils.py (hunk 3 above)
Comment (abhishekpratapa) [reply src/unity/python/turicreate/toolkits/_evaluate_utils.py]: Filed Issue: #1962

File: src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py (hunk 1 above)
Comment (abhishekpratapa) [reply src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py]: Refactored Duplicated Code

**Pull request merged**